### ISSUE TITLE
Problem with edge creation using console in distributed mode.

### ISSUE THREAD

**Issue by tobiemh** [orientechnologies/orientdb#3741]
```sql
CONNECT remote:localhost/testdatabase root root

CREATE CLASS Person extends V
CREATE CLASS Friend extends E

CREATE VERTEX Person SET fullname = 'Joe Bloggs'
CREATE VERTEX Person SET fullname = 'Tom Jones'
CREATE VERTEX Person SET fullname = 'Mark Zuckerberg'

CREATE EDGE Friend FROM ( SELECT FROM Person WHERE fullname = 'Joe Bloggs' ) TO ( SELECT FROM Person WHERE fullname = 'Tom Jones' )
CREATE EDGE Friend FROM ( SELECT FROM Person WHERE fullname = 'Joe Bloggs' ) TO ( SELECT FROM Person WHERE fullname = 'Mark Zuckerberg' )
```

Console output:

```shell
Created vertex 'Person#30:0{fullname:Joe Bloggs} v1' in 0.051000 sec(s).
Created vertex 'Person#30:1{fullname:Tom Jones} v1' in 0.040000 sec(s).
Created vertex 'Person#30:2{fullname:Mark Zuckerberg} v1' in 0.046000 sec(s).
Created edge '[Friend#33:0{out:#30:0,in:#30:1} v3]' in 0.186000 sec(s).
Created edge '[Friend#33:1{out:#30:0,in:#30:2} v3]' in 0.203000 sec(s)
```

No errors are raised on the server(s).

Querying `SELECT FROM Person` shows that the edges have been created

![screen shot 2015-03-10 at 11 43 09](https://cloud.githubusercontent.com/assets/1588290/6574223/adedff24-c71a-11e4-8abf-21cdfd39b9f2.png)

But querying `SELECT out() FROM Person` or `TRAVERSE out() FROM #30:0` returns no edges.

![screen shot 2015-03-10 at 11 43 51](https://cloud.githubusercontent.com/assets/1588290/6574230/c30db976-c71a-11e4-9d31-3394af233f4f.png)

NOTE: The same queries work using REST and/or the Orient Studio without issues.

NOTE: The same queries work in standalone mode instead of distributed mode.

***

OrientDB Version: 2.0.3 AND 2.0.4

**Comment by lvca** [comment]
Fixed in 2.0.5-SNAPSHOT and 2.1-SNAPSHOT.

**Comment by tobiemh** [comment]
I can confirm that this also works along with #3745 using 2.0.5-SNAPSHOT
